java/src/jmri/jmrix/openlcb/swing/downloader/LoaderPane.java

```diff
@@ -58,7 +58,7 @@ public void initComponents(CanSystemConnectionMemo memo) {
         this.mcs = memo.get(MemoryConfigurationService.class);
         this.dcs = memo.get(DatagramService.class);
         this.store = memo.get(MimicNodeStore.class);
-        this.nodeSelector = new NodeSelector(store);
+        this.nodeSelector = new NodeSelector(store, 10);  // display full ID
```

Comment:
What does 10 stands for? Is it any number or a specific constant? In the later case, it might be good to define a constant to express this clearly.

Reply:
10 was just larger than 4 (the max available).  I've changed it to MAX_INT and improved the comment.  Thanks!